### DISCUSSION TITLE
Cursor snapping to window edge on dragcfacts patch to make it behave the same as the dragmfacts patch

### DISCUSSION THREAD

**Discussion by cloudy-woods** [bakkeby/patches#79] [Ideas]
This would improve consistency and to me at least, them not being the same feels a bit funky to use.
These are awesome patches btw, nice work :)

**Comment by bakkeby** [comment]
I suppose it would be feasible to snap to the center of the client window, otherwise how would you know which edge to snap to, it might be counter intuitive.

Not directly relevant, but it is also possible to combine dragmfact and dragcfact into one, which gives a behaviour similar to how it works in bspwm.
